bundles/org.eclipse.osgi.services/META-INF/MANIFEST.MF

```diff
@@ -2,56 +2,26 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %osgiServices
 Bundle-SymbolicName: org.eclipse.osgi.services
-Bundle-Version: 3.10.200.qualifier
+Bundle-Version: 3.11.0.qualifier
 Bundle-Description: %osgiServicesDes
 Bundle-Localization: plugin
 Bundle-Vendor: %eclipse.org
 Bundle-DocUrl: http://www.eclipse.org
 Bundle-ContactAddress: www.eclipse.org
-Export-Package: org.osgi.service.cm;version="1.6";uses:="org.osgi.framework",
- org.osgi.service.component;version="1.4";uses:="org.osgi.framework",
- org.osgi.service.component.annotations;version="1.3",
- org.osgi.service.component.runtime;version="1.4";uses:="org.osgi.framework,org.osgi.util.promise,org.osgi.service.component.runtime.dto",
- org.osgi.service.component.runtime.dto;version="1.4";uses:="org.osgi.dto,org.osgi.framework.dto",
- org.osgi.service.device;version="1.1";uses:="org.osgi.framework",
- org.osgi.service.event;version="1.4";uses:="org.osgi.framework",
- org.osgi.service.http;version="1.2.1";uses:="javax.servlet,javax.servlet.http",
- org.osgi.service.http.context;version="1.1";uses:="org.osgi.framework,javax.servlet.http",
- org.osgi.service.http.runtime;version="1.1";uses:="org.osgi.service.http.runtime.dto",
- org.osgi.service.http.runtime.dto;version="1.1";uses:="org.osgi.dto,org.osgi.framework.dto",
- org.osgi.service.http.whiteboard;version="1.1",
- org.osgi.service.log;version="1.5";uses:="org.osgi.framework",
- org.osgi.service.metatype;version="1.4";uses:="org.osgi.framework",
- org.osgi.service.provisioning;version="1.2",
- org.osgi.service.upnp;version="1.2",
- org.osgi.service.useradmin;version="1.1";uses:="org.osgi.framework",
- org.osgi.service.wireadmin;version="1.0.1";uses:="org.osgi.framework"
-Import-Package: javax.servlet;resolution:=optional,
- javax.servlet.http;resolution:=optional,
- org.osgi.dto;version="1.0",
- org.osgi.framework;version="1.6",
- org.osgi.framework.dto;version="1.8.0",
- org.osgi.service.cm;version="[1.6,1.7)",
- org.osgi.service.component;version="[1.4,1.5)",
- org.osgi.service.component.annotations;version="[1.3,1.4)",
- org.osgi.service.component.runtime; version="[1.4,1.5)",
- org.osgi.service.component.runtime.dto; version="[1.4,1.5)",
- org.osgi.service.device;version="[1.1,1.2)",
- org.osgi.service.event;version="[1.4,1.5)",
- org.osgi.service.http;version="[1.2,1.3)",
- org.osgi.service.http.context;version="[1.1,1.2)",
- org.osgi.service.http.runtime;version="[1.1,1.2)",
- org.osgi.service.http.runtime.dto;version="[1.1,1.2)",
- org.osgi.service.http.whiteboard;version="[1.1,1.2)",
- org.osgi.service.log;version="[1.5,1.6)",
- org.osgi.service.metatype;version="[1.4,1.5)",
- org.osgi.service.provisioning;version="[1.2,1.3)",
- org.osgi.service.upnp;version="[1.2,1.3)",
- org.osgi.service.useradmin;version="[1.1,1.2)",
- org.osgi.service.wireadmin;version="[1.0,1.1)",
- org.osgi.util.function;version="[1.0,2.0)",
- org.osgi.util.promise;version="[1.0,2.0)"
-DynamicImport-Package: javax.servlet,
```

Comment:
I also removed this DynamicImport-Package which I assume is not necessary anymore.

Reply:
We don't need that if the content of this bundle doesn't have classes that require the package.